sphinx/source/docs/installation.rst

```diff
@@ -30,12 +30,20 @@ dependency on several compiled libraries.
 Ideally, you should have the following libraries installed:
 
 * NumPy
+* Pandas
 * Flask
 * Redis
+* Redis-py
+* Six
 * Requests
-* gevent
-* gevent-websocket
-* Pandas
+* Tornado >= 4.0
+* Werkzeug
+* Greenlet
+* Gevent
+* Gevent-websocket
+* PyZMQ
+* PyYaml
+* DateUtil
 
```

Comment:
Yeah, I used what was listed in the conda yaml file. I'll add the testing
dependencies as well.

I'm not sure how it is expected to work for developing on conda packages to
begin with, but I didn't see a quick way to install all requirements after
doing the setup.py develop. Since I'm in an environment, the requirements
need to be added.

Would you just install bokeh in the environment like normal, then do the
setup.py develop? I wasn't confident if the development version would then
take precedence.

On Tue, Jan 20, 2015, 7:11 AM Damián Avila <notifications@github.com> wrote:

> In sphinx/source/docs/installation.rst
> <https://github.com/bokeh/bokeh/pull/1720#discussion-diff-23220646>:
>
> >  * Requests
> > -* gevent
> > -* gevent-websocket
> > -* Pandas
> > +* Tornado >= 4.0
> > +* Werkzeug
> > +* Greenlet
> > +* Gevent
> > +* Gevent-websocket
> > +* PyZMQ
> > +* PyYaml
> > +* DateUtil
> >
>
> Oh... you did it... thanks! It would be nice to add somewhere something
> about the testing dependencies...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bokeh/bokeh/pull/1720/files#r23220646>.
>